res/TensorFlowLiteRecipes/SVDF_001/test.recipe

```diff
@@ -44,6 +44,7 @@ operation {
   input: "ifm"
   input: "weight_feature"
   input: "weight_time"
+  input: ""
```

Comment:
So, we must add empty input to recipe (as we did for FullyConnected's recipe without bias or Conv's).